locales/en-US/strings.properties

```diff
@@ -36,6 +36,8 @@ menu_action_open_new_window=Open in a New Window
 menu_action_open_private_window=Open in a New Private Window
 menu_action_dismiss=Dismiss
 menu_action_delete=Delete from History
+confirm_history_delete=Are you sure you want to delete every instance of this page from your history?
```

Comment:
I think these can be combined into one string, since they are part of the same UI message